llama_stack/log.py

```diff
@@ -156,6 +156,11 @@ def filter(self, record):
     }
     dictConfig(logging_config)
 
+    # Ensure third-party libraries follow the root log level
+    for _, logger in logging.root.manager.loggerDict.items():
+        if isinstance(logger, logging.Logger):  # Ensure it's a Logger instance
```

Comment:
nit: comments on this line and the next are not necessary. 

other than that, this looks good to me and is a good change. thank you!